saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/SemanticRoleLabeling/PopulateSRLDataModel.scala

```diff
@@ -54,7 +54,7 @@ object PopulateSRLDataModel extends Logging {
       case ViewNames.PARSE_GOLD => new ClauseViewGenerator(parseViewName, "CLAUSES_GOLD")
       case ViewNames.PARSE_STANFORD => ClauseViewGenerator.STANFORD
     }
-    def addViewAndFilter(taAll: List[TextAnnotation]): List[TextAnnotation] = {
+    def addViewAndFilter(taAll: Iterable[TextAnnotation]): Iterable[TextAnnotation] = {
```

Comment:
I like this change. You probably want to drop `toList` from [here](https://github.com/IllinoisCogComp/saul/blob/765f480d5e9d1896e319775ba0479e31a98081fc/saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/SemanticRoleLabeling/PopulateSRLDataModel.scala#L96) too.